specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,244 @@
+                "required": true,
+                "type": "string",
+                "in": "path",
+                "x-ms-skip-url-encoding": true
+            }
+        ]
+    },
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/securitydomain/download": {
+            "post": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainDownload",
+                "description": "Retrieves Security domain from HSM enclave",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "Certificates",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/CertificateSet"
+                        },
+                        "description": "Security domain download operation requires customer to provide three certificates in PEM format"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Security domain backup response",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "value": {
+                                    "type": "string"
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Key Vault error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "common.json#/definitions/KeyVaultError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Retrieve Security domain": {
+                        "$ref": "./examples/securitydomaindownloadpost-example.json"
+                    }
+                }
+            }
+        },
+        "/securitydomain/upload": {
+            "get": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainUpload",
+                "description": "Retrieve security domain transfer key",
+                "responses": {
+                    "200": {
+                        "description": "Security domain transfer key",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "transfer_key": {
+                                    "type": "string"
```

Comment:
Transfer key should be returned by REST API as a JWK. The client (CLI, PowerShell, Portal) will convert it into a .PEM.